### ISSUE TITLE
Jan - TODOS

### ISSUE THREAD

**Issue by xliucs** [ubicomplab/rPPG-Toolbox#5]
TODOs in Jan / Feb 

Deep learning models implementations - Jan - Xin 
Traditional signal processing -  - Jan  - Xiaoyu
Training DL  - Jan  -  Xin
Datasets Support - Jan - Xiaoyu
Inference DL  - Jan  - Xin
Providing ground truth labels for UBFC, MMSE, PURE  - Jan - Xiaoran
Benchmarking - Feb (pre-trained models (PURE + UBFC, Test on MMSE)) - Xin
Visualization - Jan - Xiaoyu / Xiaoran 
Synchronization - Feb - Xiaoran 
Highlight what conditions the toolbox will work well. E.g., stationary, studio light, heavily compressed etc.  - Jan - Xin

@Xiaoyu-SZ @Jiang-Xiaoran
